libraries/fabric-shim/lib/handler.js

```diff
@@ -527,17 +553,6 @@ class ChaincodeSupportClient {
             this.msgQueueHandler.queueMsg(qMsg);
         });
     }
-
```

Comment:
would prefer not to remove the toString as this will help with log files and debugging.